README.md

```diff
@@ -78,6 +78,64 @@ Since imgix can generate as many derivative resolutions as needed, ember-cli-img
 
 NB: Since this library sets [`fit`](https://docs.imgix.com/apis/url/size/fit) to `crop` by default, when just a width or height is set, the image will resize and maintain aspect ratio. When both are set, the image will be cropped to that size, maintaining pixel aspect ratio (i.e. edges are clipped in order to not stretch the photo). If this isn't desired, set `fit` to be another value (e.g. `clip`)
 
+#### Lazy Loading
+
+If you'd like to lazy load images, we recommend using [lazysizes](https://github.com/aFarkas/lazysizes). In order to use ember-cli-imgix with lazysizes, you can simply tell it to generate lazysizes-compatible attributes instead of the standard `src`, `srcset`, and `sizes` by changing some configuration settings:
+
+```jsx
+// config/environment.js
+
+module.exports = function(environment) {
+  var ENV = {
+    // snip
+    APP: {
+      imgix: {
+        attributeNameMap: {
+          src: 'data-src',
+          srcset: 'data-srcset'
+        }
+      }
+    }
+    // snip
+  };
+};
+```
+
+Otherwise, you can use the component as normal.
+
+**NB:** It is recommended to use the [attribute change plugin](https://github.com/aFarkas/lazysizes/tree/gh-pages/plugins/attrchange) in order to capture changes in the data-\* attributes. Without this, changing the attributes to this library will have no effect on the rendered image.
+
+#### Low Quality Image Placeholder Technique (LQIP)
+
+If you'd like to use LQIP images, like before, we recommend using [lazysizes](https://github.com/aFarkas/lazysizes). In order to use ember-cli-imgix with lazysizes, you can simply tell it to generate lazysizes-compatible attributes instead of the standard `src`, `srcset` by changing some configuration settings, and providing a fallback image to `placeholderSrc`.
+
+```jsx
+// config/environment.js
+
+module.exports = function(environment) {
+  var ENV = {
+    // snip
+    APP: {
+      imgix: {
+        attributeNameMap: {
+          src: 'data-src',
+          srcset: 'data-srcset'
+        }
+      }
+    }
+    // snip
+  };
+};
+```
+
+```hbs
+// usage
+
+{{imgix-image path='/abc.png' placeholderSrc='/abc.png?w=80&h=50' }}
+```
+
+**NB:** If the props of the image are changed after the first load, the low quality image will replace the high quality image. In this case, the `src` attribute may have to be set by modifying the DOM directly, or the lazysizes API may have to be called manually after the props are changed. In any case, this behaviour is not supported by the library maintainers, so use at your own risk.
```

Comment:
When you say "the library maintainers", you're talking about Lazysizes' maintainers, right? Probably want to be specific here.

Reply:
It was meant to reference us, as maintainers of ember-cli-imgix. I'll change it to be clearer.